src/mavlink_interface.cpp

```diff
@@ -397,6 +397,8 @@ void MavlinkInterface::SendGpsStatusMessages(const SensorData::GpsStatus &data)
         mavlink_message_t msg;
         mavlink_msg_gps_status_encode_chan(1, 200, MAVLINK_COMM_0, &msg, &gps_status);
         forward_mavlink_message(&msg);
+        snr_message = (char*) gps_status.satellite_snr;
+        snr_socket.sendMessage(snr_message);
```

Comment:
```suggestion
        
        snr_socket.sendMessage((void*)&(gps_status.satellite_snr), sizeof(gps_status.satellite_snr);
```
on the client side, you will have to unpack the bytes yourself which are defined in `mavlink_interface.hpp` under the GpsStatus structure